.github/workflows/localization.yml

```
@@ -0,0 +1,73 @@
+name: "localization"
+on:
+  workflow_call:
+    inputs:
+      branch_name:
+        type: string
+        description: "Branch name"
+        required: true
+        default: "master"
```

actionlint

input "branch_name" of workflow_call event has the default value "master", but it is also required. if an input is marked as required, its default value will never be used

(events)